backend/voting/tests/test_forms.py

```diff
@@ -0,0 +1,31 @@
+from api.helpers.ids import encode_hashid
+from api.voting.forms import HashidModelChoiceField
+from django.forms import ValidationError
+from pytest import mark, raises
+from submissions.models import Submission
+
+
+@mark.django_db
+def test_hashid_model_choice_field(submission):
+    field = HashidModelChoiceField(queryset=Submission.objects.all())
```

Comment:
I think we should create a custom manager named `HashidManager` that has support for hashids, in case we want to use it elsewhere too. Probably something for another PR